### DISCUSSION TITLE
Is there any guide available for migrating from neon db to local db ?

### DISCUSSION THREAD

**Discussion by balakumarane** [tgdrive/teldrive#337] [Q&A]
I want to update to the latest version but stuck with the older version because I wan't able to successfully migrate to local database.

**Comment by bpawnzZ** [comment]
No guide I know of but I did it a while back and I used pg_dump.  Then you use I think just psql to import all the data back into the new database.  A little searching around and ask AI and and u should be able to figure it out.

**Reply by balakumarane** [reply]
I have tried this and didn't work as schema got changed, after loading the server is not running. Tried to download CSV from neon and load it directly, some column name are removed and changed so it is also failure.
